### PR TITLE
[P2022-185] Header component

### PR DESCRIPTION
**Pull request by GrzegorzBlack**
Fixed version of Header component

### CODE REVIEW

File: src/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export default function PageHeader({
+  pageTitle,
+  buttonText,
+  buttonHandler = () => {},
+}: SectionProps) {
+  return (
+    <Box>
+      <StyledContainer>
+        <StyledGrid container justifyContent='space-between' height='152px'>
```
Comment (Voland91) [inline src/components/PageHeader/PageHeader.tsx:25]: You have styles in the style file, why did you add inline styles here?

**Review by Voland91 — COMMENTED** [review]
added comment

File: src/components/PageHeader/PageHeader.tsx (hunk 1 above)
Comment (GrzegorzBlack) [reply src/components/PageHeader/PageHeader.tsx]: Ok , I removed this styling to style file.

**Review by Voland91 — APPROVED** [review]
